### ISSUE TITLE
ENHANCEMENT: Support Custom Surface Exposure Types

### ISSUE THREAD

**Issue by ed-p-may** [PH-Tools/honeybee_grasshopper_ph#48]
- [PHX Issue #65](https://github.com/PH-Tools/PHX/issues/65)

**Comment by ed-p-may** [comment]
Closing as a duplicate of [PH-Tools/PHX#65](https://github.com/PH-Tools/PHX/issues/65), which is the same request and is where the work actually lives (custom exposure zones in both WUFI-Passive and PHPP-10).

This issue held no detail beyond the link. If Grasshopper-side UI work is needed once PHX#65 lands — a component for authoring the custom exposure assignment — that should be filed here as a new issue at that point.
